pkg/cache/cache.go

```diff
@@ -33,7 +35,7 @@ var (
 
 // LRUExpireCache is a cache that ensures the mostly recently accessed keys are returned with
 // a ttl beyond which keys are forcibly expired.
-type LRUExpireCache struct {
+type LRUExpireCache[V types.AuthWebhookResponse] struct {
```

Comment:
It would be nice to be able to use various values ​​as well as `AuthwebhookResponse` for Cache. Since `LRUExpireCache` does not directly manipulate the Value type, it seems that the type constraint can be set in a wide range.
```suggestion
type LRUExpireCache[V any] struct {
```